src/js/vic-v2/components/CropperController.jsx

```diff
@@ -131,13 +133,19 @@ export default class CropperControls extends React.Component {
     };
   }
 
-  componentWillUpdate(nextProps) {
-    if (nextProps.windowWidth !== this.props.windowWidth) {
-      const cropper = this.refs.cropper;
-      if (cropper) {
-        this.setCropBox();
-      }
-    }
+  componentWillMount() {
```

Comment:
It looks like using this method isn't recommended and we could put this in the constructor

https://reactjs.org/docs/react-component.html#componentwillmount

Reply:
Good catch

Reply:
This has been updated.